src/CMakeLists.txt

```diff
@@ -454,11 +450,6 @@ if (TARGET_OPENCL)
     target_compile_definitions(Halide PRIVATE WITH_OPENCL)
 endif ()
 
-option(TARGET_OPENGL "Include OpenGL/GLSL target" ON)
-if (TARGET_OPENGL)
-    target_compile_definitions(Halide PRIVATE WITH_OPENGL)
-endif ()
-
```

Comment:
Uh oh! This breaks openglcompute! That macro is used later to enable OGLC stuff.